miners.json

```diff
@@ -36,6 +36,15 @@
         "t020975",
         "t08094"
       ]
+    },
+	{
+      "name": "Asia",
+      "amount": 1,
+      "minerAddresses": [
+        "t09620",
+        "t09675",
+		"t08242"
```

Comment:
Can you submit miners via this issue:  https://github.com/filecoin-project/slingshot/issues/79  and I'll add as part of  a nightly update process.   Please remove this change from the PR.

Reply:
Okay, got it.